pom.xml

```diff
@@ -76,5 +76,10 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
MetricSet.getMetrics() has no agreement that the Map must be immutable. http://metrics.dropwizard.io/3.1.0/apidocs/com/codahale/metrics/MetricSet.html#getMetrics--

The reason for the concern is the maintenance and management of additional dependencies (enhancements, bug fixes, security concerns) on a library that will potentially be included in many other projects (understanding that Guava is a very stable library). Trying to maintain a dependency tree that has a lot of additional libraries is overhead. It is a minor concern but if it is mainly for the ease of building out one or two Maps I don't see the need.